src/data/i18n/languages.ts

```diff
@@ -0,0 +1,6 @@
+export enum Languages {
```

Comment:
Native TypeScript enums are full of problems, please avoid using them if possible.